jockeyJSAndroid/build.gradle

```diff
@@ -0,0 +1,22 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 17
+    buildToolsVersion "21.1.1"
+
+    defaultConfig {
+        minSdkVersion 8
+        targetSdkVersion 17
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
+        }
+    }
+}
+
+dependencies {
+    compile 'com.google.code.gson:gson:2.2.4'
```

Comment:
I **think** that as long as we define the dependency we are good. Also, if every aar/jar had to bundle it's own dependencies with itself things would quickly get out of hand, so I suspect it's unnecessary.